README.md

```diff
@@ -1 +1,218 @@
-Magento_UpwardConnector module used for correctly routing frontend requests through UPWARD-PHP.
+# How to install UPWARD with a PWA Studio storefront
+
+## Prerequisites
+
+- A dev, starter, or pro tier Magento Cloud project
+- Composer
+- Yarn or NPM
+- A [Magento PWA Studio][] storefront managed by NPM or Yarn.
+
+  You are not required to publish to npmjs.com, but
+  NPM or Yarn should be able to access your project code through Git.
+
+  For this tutorial, the [`venia-concept`][] package for the [Venia storefront][] is used, but any PWA available though NPM with an UPWARD compatible YAML file is supported.
+
+## Add node package dependency
+
+If it does not exist, [create a `package.json`][] file in your project directory.
+
+```sh
+npm init -y
+```
+
+Add your PWA Studio storefront to the list of dependencies in this file.
+
+```json
+{
+  "dependencies": {
+    "@magento/venia-concept": "~2.0.0"
+  }
+}
+```
+
+## Add Composer dependencies
+
+Add the repository information for the UPWARD PHP server and UPWARD Connector to the `repositories` section in your `composer.json`.
+
+```json
+"upward-connector": {
+    "type": "vcs",
+    "url": "https://github.com/magento-research/magento2-upward-connector"
+},
+"upward-php": {
+    "type": "vcs",
+    "url": "https://github.com/magento-research/upward-php"
+}
+```
+
+To have composer install these packages, add them to the `require` section of the `composer.json` file.
+
+```json
+"magento/module-upward-connector": "^1.0.0",
+"magento/upward": "^1.0.0"
+```
+
+<!--
+* TODO: change `dev-test-cloud-integration` to `dev-master` when it is merged
+* TODO: Remove instructions for adding repository locations after they have been published on Packagist.org and/or packages.magento.com
+* TODO: After the packages are published, remove any mention of specifying a version for `magento/upward` as it will be included with `magento/module-upward-connector`
+* TODO: After publishing, make sure the version of `magento/module-upward-connector` matches the actual version i.e. `^2.3` or `*`
+-->
+
+## Add Venia sample data (optional)
```

Comment:
We still have https://github.com/magento-research/pwa-studio/blob/release/2.0/packages/venia-concept/deployVeniaSampleData.sh and it still works! You might want to verify that, but we could recommend this script to automate the next couple steps.

Reply:
Added option to use the script in that package